epochtalk/epochtalk
# Deletion

Thread Deletion

Testing Procedure:
* Login as a Admin/Mod
* Create a thread
  * Ensure the thread and post count on the board is incremented
* Delete a thread 
  * Ensure that the thread and post count on the board is decremented
* Undelete a thread
  * Ensure that the thread and post count on the board is incremented again
* Purge a thread
  * Ensure the that thread and post count on the board is decremented again

* Login as a regular user
* Create a thread
* Ensure that you cannot delete, undelete or purge a thread

This pull request is dependent on the same branch in the core-pg repo.
